### PR TITLE
Added Plane Python interface

### PR DESCRIPTION
**Pull request by ahcorde**
Signed-off-by: ahcorde <ahcorde@gmail.com>

# 🎉 New feature

This PR is part of this meta ticket https://github.com/ignitionrobotics/sdformat/issues/931

## Summary

Added Plane Python interface

## Test it

```python
from sdformat import Plane

plane = Plane()
```

## Checklist
- [x] Signed all commits for DCO
- [x] Added tests
- [ ] Added example and/or tutorial
- [ ] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [x] `codecheck` passed (See [contributing](https://ignitionrobotics.org/docs/all/contributing#contributing-code))
- [x] All tests passed (See [test coverage](https://ignitionrobotics.org/docs/all/contributing#test-coverage))
- [x] While waiting for a review on your PR, please help review [another open pull request](https://github.com/pulls?q=is%3Aopen+is%3Apr+user%3Aignitionrobotics+repo%3Aosrf%2Fsdformat+archived%3Afalse+) to support the maintainers

**Note to maintainers**: Remember to use **Squash-Merge** and edit the commit message to match the pull request summary while retaining `Signed-off-by` messages.

### CODE REVIEW

**Comment by ahcorde** [comment]
@osrf-jenkins retest this please

**Comment by ahcorde** [comment]
@osrf-jenkins retest this please

**Review by scpeters — COMMENTED** [review]
I will apply this change

File: python/src/sdf/pyPlane.cc [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    .def("size", &sdf::Plane::Size,
+         "Get the plane size in meters.")
+    .def("set_size", &sdf::Plane::SetSize,
+         "Set the plane size in meters.")
+    .def("normal", &sdf::Plane::Normal,
+          "Get the plane normal vector. When a Plane is used as a geometry "
+          "for a Visual or Collision object, then the normal is specified in "
+          "the Visual or Collision frame, respectively.")
+    .def("set_normal", &sdf::Plane::SetNormal,
+          "Set the plane normal vector. The _normal vector will be "
```
[the remark this answers is not in the document]
Comment (scpeters) [reply python/src/sdf/pyPlane.cc]: https://github.com/ignitionrobotics/sdformat/pull/941/commits/5015a7e12238959f3a2088b5a5b4695469a6250a

**Review by scpeters — APPROVED** [review]

**Pull request merged**